department-of-veterans-affairs/vets-api
# Update ruby version for k8s

## Summary

This PR updates the Ruby version to the latest patch (3.2.3) for the app as well as the Dockerfile. See [release notes](https://github.com/ruby/ruby/releases/tag/v3_2_3) for what was fixed in the patch version.

## Related issue(s)

- *Link to ticket created in va.gov-team repo OR screenshot of Jira ticket if your team uses Jira*
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

* Update ruby version locally.
* Verify app runs as expected.
* Verify all checks pass.


## Screenshots
Not relevant.

## What areas of the site does it impact?
Everything?

## Acceptance criteria


## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
